BccMembers.Api.Client2/BccMembersApiClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Net.Http;
```

Comment:
Seems like there are a couple of redundant files here in Client2.

Reply:
Do you mean usings @tkristiansen? All usings are currently used

Reply:
right, I see